backend/os/location.go

```diff
@@ -19,15 +19,13 @@ type Location struct {
 	fileSystem vfs.FileSystem
 }
 
-// NewFile uses the properties of the calling location to generate a vfs.File (backed by an os.File). A string
+// NewFile uses the properties of the calling location to generate a vfs.File (backed by a osFile). A string
```

Comment:
Why was this changed?

Reply:
no idea. I must have changed it at some point and i couldn't remember whether the period should have been in there or not.